scripts/hid_configurator/requirements.txt

```diff
@@ -1,3 +1,3 @@
 # pyhidapi
-hid>=1.0.3, <1.0.5
+hid==1.0.8
```

Comment:
I would desist from specifying the library version (remove the upper version limit). It seems that the issue was caused by incompatibilities between Python `hid` package version and version of the underlying libraries. This eventually triggered: `undefined symbol: hid_get_input_report`.

Requiring newer version here might cause problems for users that need to use older version of underlying libraries (e.g. older OS release). Requiring older version might lead to missing some updates (theoretically including security fixes).